### ISSUE TITLE
Ministers module

### ISSUE THREAD

**Issue by tb0yd** [tb0yd/civplicity#1]
Add a pluggable module to add "Ministers" to the game.

TL;DR: Ministers would combine an alert system for game events, help system for newer players, and intelligence on other civs, while also adding more depth to the AI.

Ministers would appear randomly from your cities according to how much culture they are producing and spit out auto-generated advice for you to read. Ministers each have their own victory mode, which is random and highly influenced by your actions as leader. For example, if you play like a Space Racer, 90% the Ministers you get would remind you of the amazing marvels of science in a nearby highly-populated enemy city, and urge you to go invade it.

Their knowledge is local to their city both in choosing their own victory mode and giving you specific advice. They are only aware of how you run their home city, and the orders you give to units produced by and stationed in that city, as well as foreign activity immediately around that city and your diplomatic status with other players. Ministers also influence one another while working for you, so if you have 5 who are Dominators and 1 who's a Culturer, pretty soon that Culturer will turn into a Dominator.

As a fun twist, if you conquer an opponent's city, it will start producing Ministers for you after about 30 years, and those Ministers will be more influenced by the previous player's actions than your actions for a longer period of time, depending on how long that previous player's rule was. So if you're a Space Racer and you conquer a city that was ruled by a Dominator for millenia, Ministers from there will be Dominators for centuries no matter what the other Ministers think. This gives you valuable insight into other players' strategy.

AI players could have Ministers too, and their actions would be partially based on the Ministers they get. 

Examples: 
- "Jack O'Hare" of Boston (8), Diplomat, American, year 1803CE. Notice the advice comes off as paranoid and is all about other nations. This would be the Diplomat style of advice.

```
Imminent disaster is at hand. I have heard rumors that the Canadians are preparing a surprise attack against the Iroquois capitol of Onondaga. May I remind you that this action would throw the Iroquois into Anarchy and expel both nations from the UN! I highly suggest producing a legion of Cavalry to cut them off at the pass. The future of the world is at stake. And keep a close eye on the British because they are a treacherous people.
```
- "Yusuf al-Ghamidi" of Marrakech (3), Dominator, Moroccan, year 2400BCE. There aren't many other players on the map, but the player has mostly been producing Swordsmen and Archers. He hints at researching Astronomy because that's a key tech for Dominators (lets them get to capitols more quickly).

```
Our army is unstoppable! Send a strong delegation south, where I believe there are many tribes of barbarians. Bring them under our flag with a swift and decisive offensive. We are strong, but honorable. My father used to tell me tales of people who were carried to far-off lands across the ocean by watching the stars. Have you heard of this?
```
